include/ur.py

```diff
@@ -1902,6 +1902,9 @@ class ur_function_v(IntEnum):
     BINDLESS_IMAGES_WAIT_EXTERNAL_SEMAPHORE_EXP = 148   ## Enumerator for ::urBindlessImagesWaitExternalSemaphoreExp
     BINDLESS_IMAGES_SIGNAL_EXTERNAL_SEMAPHORE_EXP = 149 ## Enumerator for ::urBindlessImagesSignalExternalSemaphoreExp
     PLATFORM_GET_LAST_ERROR = 150                   ## Enumerator for ::urPlatformGetLastError
+    USM_P2_P_ENABLE_PEER_ACCESS_EXP = 151           ## Enumerator for ::urUsmP2PEnablePeerAccessExp
```

Comment:
"P2P" is being translated to "P2_P" here and in various other places. I was thinking of getting around this by switching to "usm-ptop", but this is not ideal since I don't think I am allows the case variation: "PtoP"?

Reply:
We've merged a fix. You can now rebase, and this issue will be gone.